src/main/resources/assets/farmersdelight/lang/lzh.json

```
@@ -0,0 +1,322 @@
+{
+  "block.farmersdelight.stove": "灶",
+  "block.farmersdelight.cooking_pot": "鼎",

```

`鼎` more used as a ritual vessel, and its size might be too large. `鑊` might be better.
`鼎`更多是用作一种礼器，且形制也大很多，建议使用`鑊`
```suggestion
  "block.farmersdelight.cooking_pot": "鑊",

```